app/domain/structured_extraction.py

```
@@ -0,0 +1,77 @@
+        # instructor will handle forcing the StructuredDocumentData schema
+        structured_data = client.chat.completions.create(
+            model=full_model_name,
+            messages=[
+                {"role": "system", "content": "You are a precise data extraction AI."},
+                {"role": "user", "content": prompt}
+            ],
+            response_model=StructuredDocumentData,
+            max_tokens=4096,
+        )
+        return structured_data
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)